gcn/notices/icecube/single_neutrino_alerts.example.json

```diff
@@ -1,10 +1,17 @@
 {
-  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/icecube/test/gold_bronze_track_alerts.schema.json",
-  "additional_info": "IceCube Bronze Neutrino Track Alert",
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/icecube/single_neutrino_alerts.schema.json",
+  "mission": "IceCube",
+  "instrument": "IC86",
+  "messenger": "Neutrino",
+  "notice_type": "Bronze Track Alert",
+  "record_number": 0,
```

Comment:
Would you mind starting this count from 1 rather than 0?  We haven't been explicit in our description, but that's the general usage.